tests/test_stats/prediction/prediction_with_encoder.json

```diff
@@ -2,127 +2,107 @@
   "eopatch-id-0-col-0-row-0": {
     "bbox": "BBox(((729480.0, 4390045.0), (732120.0, 4391255.0)), crs=CRS('32638'))",
     "data_timeless": {
-      "PREDICTION_PROBABILITY": [
-        121,
-        264,
-        3,
-        "float64",
-        0,
-        95832,
-        0.00514,
-        0.77911,
-        0.33333,
-        0.21575,
-        31944,
-        0,
-        31944,
-        0,
-        0,
-        0,
-        0,
-        31944,
-        0.77911,
-        0.21575,
-        0.21575,
-        0.77911,
-        0.77911,
-        0.00514,
-        0.21575,
-        0.77911
-      ]
+      "PREDICTION_PROBABILITY": {
+        "array_shape": [
+          121,
+          264,
+          3
+        ],
+        "dtype": "float64",
+        "random_values": [
+          0.77911,
+          0.21575,
+          0.21575,
+          0.77911,
+          0.77911,
+          0.00514,
+          0.21575,
+          0.77911
+        ],
+        "values": [
+          {
```

Comment:
I'll trust this is correct :)

Reply:
It is a bit surprising but I think it should be correct.